#! /bin/sh /usr/share/dpatch/dpatch-run
## disable-ifstat.dpatch by Andreas Henriksson <andreas@fatal.se>
## Original author unknown, these changes where split out to a dpatch
## from a direct source edit.
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disable building ifstat. Reason unknown.
## DP: Possibly to "solve" the file collision with the
## DP: ifstat package, which also ships /usr/bin/ifstat.

@DPATCH@
diff -urNad pkg-iproute~/misc/Makefile pkg-iproute/misc/Makefile
--- pkg-iproute~/misc/Makefile	2008-01-09 21:00:54.000000000 +0100
+++ pkg-iproute/misc/Makefile	2008-01-09 21:02:42.000000000 +0100
@@ -1,7 +1,8 @@
 SSOBJ=ss.o ssfilter.o
 LNSTATOBJ=lnstat.o lnstat_util.o
 
-TARGETS=ss nstat ifstat rtacct arpd lnstat
+#TARGETS=ss nstat ifstat rtacct arpd lnstat
+TARGETS=ss nstat rtacct lnstat arpd
 
 include ../Config
 
